datasets/br/br_slavery/br_slavery.yml

```diff
@@ -0,0 +1,21 @@
+title: Brazil Ministry of Labour and Employment slavery prevention list
+entry_point: crawler.py
+prefix: br-slavery
+coverage:
+  frequency: weekly
+  start: 2025-01-08
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  A list of employers who have submitted workers to conditions analogous to that of a slave.
+publisher:
+  name: Ministry of Labour and Employment of Brazil
```

Comment:
we now have a name_en field here - not sure it's in docs yet - it'd be nice to put the brazilian name in `name` and the translation in `name_en`